docs/designs/crd.md

```diff
@@ -599,19 +599,58 @@ Monitoring defines Opensearch monitoring configuration
         <td>false</td>
         <td>admin</td>
       </tr><tr>
-        <td><b>ScrapeInterval</b></td>
+        <td><b>scrapeInterval</b></td>
         <td>string</td>
         <td>Define interval for scraping</td>
         <td>false</td>
         <td>30s</td>
       </tr><tr>
       </tr><tr>
-        <td><b>PluginURL</b></td>
+        <td><b>pluginURL</b></td>
         <td>string</td>
         <td>Define offline link to Aiven Plugin</td>
         <td>false</td>
         <td>https://github.com/aiven/prometheus-exporter-plugin-for-opensearch/releases/download/<YOUR_CLUSTER_VERSION>/prometheus-exporter-<YOUR_CLUSTER_VERSION>.zip/</td>
       </tr><tr>
+      </tr><tr>
+        <td><b>scheme</b></td>
+        <td>string</td>
+        <td>HTTP scheme to use for scraping</td>
+        <td>false</td>
+        <td>https</td>
+      </tr><tr>
+      </tr><tr>
+        <td><b>tlsConfig</b></td>
+        <td>map[]</td>
+        <td>Tls Configuration</td>
+        <td>false</td>
+        <td> - </td>
+        <table>
```

Comment:
There is some mixup and the tlsconfig table is defined inside the other table.

Reply:
Moved it out, hopefully it is ok